src/session-pool.ts

```diff
@@ -1036,6 +1151,71 @@ export class SessionPool extends EventEmitter implements SessionPoolInterface {
     session.txn = transaction;
   }
 
+  /**
+   * recycles sessions with long-running transactions
```

Comment:
Are we recycling the sessions (as per this comment)? Or are we deleting them (as per the function name)? Or are we removing them from the session pool?

Reply:
We recycle the sessions and delete the transaction. Maybe a better name should be used